core/services/eth/client.go

```diff
@@ -24,6 +24,7 @@ import (
 )
 
 //go:generate mockery --name Client --output ../../internal/mocks/ --case=underscore
+//go:generate mockery --name Client --output mocks/ --case=underscore
```

Comment:
I believe we're generally trying to move to having mocks per package, starting that for the eth package here